packages/experiments/src/components/Pagination/Pagination.types.ts

```diff
@@ -101,14 +101,14 @@ export interface IPaginationProps {
 
 export enum PaginationFormat {
```

Comment:
Another way to define this which removes the need for an enum is to do 

```ts
export type PaginationFormat = 'comboBox' | buttons';
```

Which you might consider as part of this PR or a future PR.

Reply:
yes, this is certainly a change we'd want made before major release. either in this PR or make note in the issue.

Reply:
sure. Updated in this pr

---
In reply to: [242258511](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r242258511) [](ancestors = 242258511)

Reply:
updated in this pr

---
In reply to: [241932420](https://github.com/OfficeDev/office-ui-fabric-react/pull/7411#discussion_r241932420) [](ancestors = 241932420)

Reply:
Awesome, thank you 😃!